docs/kyma-release-notes/2023-11-08-release-notes-2.20.md

```diff
@@ -0,0 +1,19 @@
+Colorful leaves dropping on the ground signify high autumn. With Kyma 2.20, we say goodbye to some of our components that fall down the Kyma tree. However, it is a natural process as they will be reborn in their modular versions, and we will watch them in full bloom again. Read on to learn more about changes in the new Kyma release.
+
+## Application Connector
+
+### Removal of the Application Connector component
```

Comment:
We don't remove the component, we migrated it to a module, keeping functionality and providing more benefits. That is a big difference to the monitoring removal were we actually removed functionality

Reply:
What Otters and Frogs probably meant was removing AC and Serverless from the components list and that is in a sense "removal". However, the sense is definitely different compared to the ultimate Monitoring component removal. @kwiatekus @koala7659 can we rephrase "removal" to "migration" in the case of AC and Serverless?

Reply:
Yes. Migration sounds better. Thank you